lib/msal-browser-poc/package.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "msal-browser",
+    "author": {
+        "name": "Microsoft",
+        "email": "nugetaad@microsoft.com",
+        "url": "https://www.microsoft.com"
+    },
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
+    },
+    "version": "0.0.1",
+    "description": "Microsoft Authentication Library for js",
+    "keywords": [
+        "implicit",
+        "authorization code",
+        "PKCE",
+        "js",
+        "AAD",
+        "msal",
+        "oauth"
+    ],
+    "main": "./dist/index.js",
+    "module": "./dist/index.es.js",
+    "types": "./dist/index.d.ts",
+    "engines": {
+        "node": ">=0.8.0"
```

Comment:
We should change this to whatever version is run in CI

Reply:
Ok is there a recommendation for what version we should be using? In CI we are just using latest stable.